Fly_Swatter/graph_trajectory.py

```diff
@@ -5,6 +5,37 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
+def plot_radar(x,y,z):
```

Comment:
new plot of a radar display like screen; going to want to add multiple target capabilities in the future and perhaps another screen with vector